frontend/src/components/CheckboxSelect.js

```diff
@@ -48,6 +48,7 @@ export default function CheckboxSelect({
   disabled,
   hideToggleAll,
   onChange,
+  previousValue,
```

Comment:
Can we use something like https://blog.logrocket.com/how-to-get-previous-props-state-with-react-hooks/ so the user of `CheckboxSelect` don't have to worry about passing in the previous value explicitly?